server/server/src/client.ts

```diff
@@ -154,7 +158,9 @@ export class ClientSession implements Session {
     query: DocumentQuery<T>,
     options?: FindOptions<T>
   ): Promise<void> {
-    await ctx.sendResponse(ctx.requestId, await this.findAllRaw(ctx.ctx, ctx.pipeline, _class, query, options))
+    await this.findRateLimit.exec(async () => {
```

Comment:
Why we need it here? It's really bad....

Reply:
Why bad? To prevent ddos by individual connection.